runtime/run.sh

```diff
@@ -30,17 +37,21 @@ if [[ ! -c /dev/net/tun ]]; then
   mknod /dev/net/tun c 10 200
 fi
 
-echo "Waiting for leader election..."
-LEADER=false
-while [[ "${LEADER}" == "false" ]]; do
-  CURRENT_LEADER=$(curl http://127.0.0.1:4040 -s -m 2 | jq -r ".name")
-  if [[ "${CURRENT_LEADER}" == "$(hostname)" ]]; then
-    echo "I am the leader."
-    LEADER=true
-  else
-    sleep 1
-  fi
-done
+if [[ "${DEBUG_SKIP_LEADER}" == "true" ]]; then
+  echo "CAUTION: Skipping leader election due to DEBUG_SKIP_LEADER==true."
+else
+  echo "Waiting for leader election..."
+  LEADER=false
+  while [[ "${LEADER}" == "false" ]]; do
+    CURRENT_LEADER=$(curl http://127.0.0.1:4040 -s -m 2 | jq -r ".name")
+    if [[ "${CURRENT_LEADER}" == "$(hostname)" ]]; then
+      echo "I am the leader."
+      LEADER=true
+    else
+      sleep 1
+    fi
+  done
```

Comment:
In bash, all regular `break` to go out of the loop works. So you can do:
```suggestion
  while :; do
    CURRENT_LEADER=$(curl http://127.0.0.1:4040 -s -m 2 | jq -r ".name")
    if [[ "${CURRENT_LEADER}" == "$(hostname)" ]]; then
      echo "I am the leader."
      break
    sleep 1
  done
```
Note: `:` is a noop - but alternatively, if you find it more readable, you can use `while true;`